modules/rubiconBidAdapter.js

```diff
@@ -267,6 +267,15 @@ export const spec = {
         utils.deepSetValue(data, 'source.ext.schain', bidRequest.schain);
       }
 
+      /**
+       * Prebid AdSlot
+       * @type {(string|undefined)}
+       */
+      const pbAdSlot = utils.deepAccess(bidRequest, 'context.pbAdSlot');
+      if (typeof pbAdSlot === 'string' && pbAdSlot) {
+        utils.deepSetValue(data.imp[0], 'context.data.adslot', pbAdSlot);
```

Comment:
noticed that in *prebidServierBidAdapter* this is  `ext.context.data.adslot`.  why are they different?

Reply:
fixed